### DISCUSSION TITLE
[POLL] Keira3 usage survey

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3918] [General]
I would like to collect your feedback about the usage of Keira3. You can help by answering the questions below:

1) How do you use Keira3? What features do you use the most? How frequently?

2) Do you use other DB editors to write your SQL code? If yes, what features are missing or not good enough in Keira3?

3) Which ones of the current features do you think needs to be improved in order to make your development experience better? (excl. 2)

4) What other features do you think should be part of Keira3? (excl. 2)

5) Any other comment that doesn't fit in the previous questions, is welcome here.

**Comment by Kitzunu** [comment]
1. SAI editor only
2. Not really, I just prefer doing anything but SAI by hand
3. For the dropdown menu for SAI Event and SAI Action should be possible to enter number value manually or using the dropdown/search for one so you don't have to scroll and search for the right one. (Same for condition edit I suppose)
4. N/A
5. Great work :)

**Comment by FrancescoBorzi** [comment]
> For the dropdown menu for SAI Event and SAI Action should be possible to enter number value manually or using the dropdown/search for one so you don't have to scroll and search for the right one. (Same for condition edit I suppose)

You can already do it, just select the dropdown and start typing your number :) It works like any html5 select dropdown

**Comment by Jinnaix** [comment]
1. Mostly SAI, a little for creature flags and loot tables.
2. No, most other editors are outdated by now and dont fit current AC/TC Databases.
3. A editor Option to change SAI IDs. Right now, if you made a mistake, you need to manually change the ID in your query/database. ![image](https://cdn.discordapp.com/attachments/682994561177747515/694139588637229107/unknown.png)
4. N/A
5. Thank you very much for this Editor, a great tool :)

**Comment by FrancescoBorzi** [comment]
@Jinnaix done: https://github.com/azerothcore/Keira3/pull/557/files

will be available in the next release
